recipes/igwn-lldd-common/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "igwn-lldd-common" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/igwn-lldd-common-0.1.0.tar.gz
+  sha256: fa70188e6185e807c92fd10f2b82bcd6046435148aebda18fb3a0f8b419a6ef9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vvv
+  run_exports:
+    - {{ pin_subpackage("igwn-lldd-common", max_pin="x") }}
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+  run:
+    - python >=3.6
+    - python-confluent-kafka
+    - kafka-python
+    - configargparse
+    - ujson
+    - gpstime
+    - watchdog
+
+test:
+  requires:
+    - pip
+    - pytest
+    - pytest-cov
+    - flaky
+  commands:
+    # check metadata
+    - python -m pip check
+    # run test suite
+    - python -m pytest --cov=igwn_lldd_common -ra -v --pyargs igwn_lldd_common.tests
+    # check entry points
+    - python -m coverage run --append --source=igwn_lldd_common -m igwn_lldd_common.frame2kafka --help
+    - python -m coverage run --append --source=igwn_lldd_common -m igwn_lldd_common.kafka2frame --help
+    # print out the coverage
+    - python -m coverage report
+    # put coverage in a xml file
+    - python -m coverage xml -o coverage-conda.xml
```

As above, measuring coverage is unnecessary (also use the entry points)

```suggestion
    - python -m pytest -ra -v --pyargs igwn_lldd_common.tests
    # check entry points
    - frame2kafka --help
    - kafka2frame --help
```